### ISSUE TITLE
Kate - QUESTION RE: "Shipibo" and "Shipibo Conibo"

### ISSUE THREAD

**Issue by stellasilbert** [theglobaljukebox/cantometrics#44]
<google-sheets-html-origin><style type="text/css"></style>

14026 |   | Why are "Shipibo" (GJB 25422) and "Shipibo-Conibo" (14026) different cultures? Is it because of the specific notes associated with recordings, which assigned different culture names to them (Shipibo in one case, Shipibo-Conibo in the other?). If the distinction is intentional, the maintenance of this level of detail seems justified, but note that eHRAF and D-PLACE currently group both Shipibo and Conibo under one id. For now,  GJB Shipibo and GJB Shipibo-Conibo are both matched to EA Se9 / eHRAF SE26. Wikipedia says: "The Shipibo-Conibo are an indigenous people along the Ucayali River in the Amazon rainforest in Peru. Formerly two groups, the Shipibo and the Conibo, they eventually became one distinct tribe through intermarriage and communal ritual and are currently known as the Shipibo-Conibo people." Note however that the Peruvian census appears to record Shipibo and Conibo as distinct groups as recently as 1974 (from eHRAF Culture Summary - Demography for Shipibo-Conibo: "A census in 1974 reported the existence of 9,000 Shipibo and another 6,000 Conibo.") And, GJB songs were all recorded in 1953 (Harry Tschopik, Jr.). So, perhaps leaving "as is" (i.e., as two cultures) makes sense (and is  most truthful to the original recording info).
-- | -- | --
25422 |   |

**Comment by stellasilbert** [comment]
Stella's notes:

1) Liner notes and tape notes describe performers of all songs currently associated with society 14026 as Conibo, so I have changed society name to Conibo. All songs associated with 25422 were identified by notes as Shipibo performers, so no changes were needed there. Added to alternate names for both cultures: Shipibo-Conibo and Chama (an alternate name for the conglomerate of the two tribes mentioned in liner notes). In terms of matching, should both still be matched to EA Se9 / eHRAF SE26? Also, for language, while glottolog lists Shipibo and Conibo as separate dialects of Shipibo-Conibo, wikipedia says that the two have merged. Since D-PLACE has the language as Shipibo-Conibo for EA Se9, I've made that the language for both GJB societies.

2) Songs 1996 and 1997 have incorrect metadata based on tape notes and original coding sheets, so I fixed that as well. Audio seems to be correct.
